ACHNBrowserUI/ACHNBrowserUI/views/todayDashboard/TodayVillagerVisitsSection.swift

```diff
@@ -0,0 +1,140 @@
+//
+//  TodayVillagerVisitsSection.swift
+//  ACHNBrowserUI
+//
+//  Created by Renaud JENNY on 06/06/2020.
+//  Copyright © 2020 Thomas Ricouard. All rights reserved.
+//
+
+import SwiftUI
+import SwiftUIKit
+import Backend
+import UI
+
+struct TodayVillagerVisitsSection: View {
+    @EnvironmentObject private var collection: UserCollection
+    @EnvironmentObject private var subManager: SubscriptionManager
+    @Binding var sheet: Sheet.SheetType?
+
+    private var residents: [Villager] { collection.residents }
+    private var visitedResidents: [Villager] { collection.visitedResidents }
+    private var rows: Int { Int((Double(residents.count)/4).rounded(.up)) }
+
+    var body: some View {
+        Section(header: SectionHeaderView(text: "Villager Visits", icon: "person.crop.circle.fill.badge.checkmark")) {
+            if residents.count > 0 {
+                villagerVisits
+            } else {
+                Text("Who are your residents? Go find your villagers and tap on the Home icon in them details to track visits here!")
+                .foregroundColor(.acText)
+                .padding(.vertical, 8)
+            }
+        }
+    }
+
+    private var villagerVisits: some View {
+        VStack(spacing: 15) {
+            bubbles
+            Text("Long press on a villager to see more info about them")
```

Comment:
```suggestion
            Text("Long press on a villager to see more info about him")
```

Reply:
`them` is gender neutral, I think it's ok here no?
Otherwise `him/her` will be better right?

Reply:
To be honest I don't know gender neutral rules in English. Just thought I sounds weird but I googled it and yes, gender neutral is better so leave it as it is

Reply:
I would keep `them`, it's the neutral form. They, them, theirs. All neutral. Animal Crossing uses these as well.